pkgs/servers/web-apps/discourse/update.py

```diff
@@ -8,17 +8,22 @@
 import re
 import logging
 import subprocess
-import pathlib
```

Comment:
This breaks `update`, line 169 also needs `pathlib.Path` -> `Path`

Reply:
Oh, whoops. Fixed it and added the latest update to `2.7.5`.